lms/static/sass/multicourse/_courses.scss

```diff
@@ -257,160 +333,152 @@ $facet-background-color: #007db8;
     }
   }
 
+}
 
-
-  .search-facets{
+// +Facets - Sidebar - Search
+// ====================
+.find-courses .search-facets {
+  @include fill-parent();
+  @include omega();
+  @include box-sizing(border-box);
+  @extend %ui-depth1;
+  position: relative;
+  margin: ($baseline*2) 0 ($baseline*3.5) 0;
+  box-shadow: 1px 2px 5px $black-t0;
+  border-top: 1px solid $black;
+  border-bottom: 2px solid $black;
+  background-color: $white;
+  max-height: ($baseline*100);
+
+  @include media($bp-tiny) {
     @include fill-parent();
-    @include omega();
-    @include box-sizing(border-box);
-    @extend %ui-depth1;
-    position: relative;
-    margin: ($baseline*2) 0 ($baseline*3.5) 0;
-    box-shadow: 1px 2px 5px $black-t0;
-    border-top: 1px solid $black;
-    border-bottom: 2px solid $black;
-    background-color: $white;
-    max-height: ($baseline*100);
+  }
 
-    @include media($bp-tiny) {
-      @include span-columns(4);
-    }
+  @include media($bp-small) {
+    @include span-columns(4);
+  }
 
-    @include media($bp-small) {
-      @include span-columns(3);
-    }
+  @include media($bp-medium) {
+    @include span-columns(8);
+  }
 
-    @include media($bp-medium) {
-      @include span-columns(4);
-    }
+  @include media($bp-large) {
+    @include span-columns(4);
+  }
 
-    @include media($bp-large) {
-      @include span-columns(4);
-    }
+  @include media($bp-huge) {
+    @include span-columns(3);
+  }
 
-    @include media($bp-huge) {
-      @include span-columns(3);
-    }
+  &.phone-menu {
+    border: medium none;
+    padding: 0;
+    overflow: visible;
+  }
 
-    &.phone-menu {
-      border: medium none;
-      padding: 0;
-      overflow: visible;
-    }
+  &:before {
+    @include right(0);
+    position: absolute;
+    top: (-$baseline*0.15);
+    opacity: 0;
+    background-color: $white;
+    padding: ($baseline*2) ($baseline*0.75) 0 ($baseline*0.75);
+    width: ($baseline*2.5);
+    height: ($baseline/4);
+    content: "";
+  }
 
-    &:before {
-      @include right(0);
-      position: absolute;
-      top: (-$baseline*0.15);
-      opacity: 0;
-      background-color: $white;
-      padding: ($baseline*2) ($baseline*0.75) 0 ($baseline*0.75);
-      width: ($baseline*2.5);
-      height: ($baseline/4);
-      content: "";
-    }
+  .header-search-facets, .header-facet {
+    @extend %t-title6;
+    padding: ($baseline/2);
+    margin: 0;
+    color: $facet-text-color;
```

Comment:
Is this variable too specific? Should it be related to action colors or base text color?